versions.def

```diff
@@ -6,7 +6,7 @@ APR_VERSION		:= 1.5.2
 
 APR_UTIL_VERSION	:= 1.5.4
 
-ASPELL_VERSION		:= 0.60.6.1
+ASPELL_VERSION		:= 0.60.7-rc1
```

Comment:
Just a question.  Are we happy pointing at a release candidate version rather than a stable one?

Reply:
Unfortunately they haven't release the stable version yet. Reference available here http://aspell.net/